lobby/src/main/java/games/strategy/engine/lobby/server/ModeratorController.java

```diff
@@ -99,7 +101,7 @@ private User getUserForNode(final INode node) {
   @VisibleForTesting
   static String getUsernameForNode(final INode node) {
     // usernames may contain a " (n)" suffix when the same user is logged in multiple times
-    return node.getName().split(" ")[0];
+    return Iterables.get(Splitter.on(' ').split(node.getName()), 0);
```

Comment:
I'd actually use a simple combination of substring and indexOf instead, but this works too I guess.

Reply:
ea5926e08